tests/unit/test_lookups.py

```diff
@@ -0,0 +1,230 @@
+import pytest
+import os
+import csv
+
+from digital_land.pipeline import Lookups
+
+
+@pytest.fixture
```

Comment:
unit tests should not be reliant on a file system as a dependency. Either it should be an integration test or by mocking.